easy-random-core/src/test/java/org/jeasy/random/EasyRandomTest.java

```diff
@@ -232,6 +232,7 @@ void nextEnumShouldNotAlwaysReturnTheSameValue() {
         assertThat(distinctEnumBeans.size()).isGreaterThan(1);
     }
 
+    @Disabled
```

Comment:
Should this be disabled? I would expect either the test to be removed, or the test to function. Or at least, an explanation why the test should exist but be disabled.